jobs/regular/check_akismet_post_voting_comment.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+module Jobs
+  class CheckAkismetPostVotingComment < ::Jobs::Base
+    def execute(args)
+      return unless SiteSetting.akismet_enabled?
+      return unless comment = PostVotingComment.find_by(id: args[:comment_id])
+      return if Reviewable.exists?(target: comment)
+
+      DistributedMutex.synchronize("akismet_post_voting_comment_#{comment.id}") do
+        if comment.custom_fields[DiscourseAkismet::Bouncer::AKISMET_STATE] == "pending"
```

Comment:
Can you create a constant somewhere in the plugin for `"pending"` so that you don't have these strings hanging around? It's used in other files too so would be nice :)